internal/librarian/release_please_lite.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package librarian
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/googleapis/librarian/internal/config"
+	"github.com/googleapis/librarian/internal/gitrepo"
+)
+
+// GetConventionalCommitsSinceLastRelease returns all conventional commits for the given library since the
+// version specified in the state file.
+func GetConventionalCommitsSinceLastRelease(repo *gitrepo.LocalRepository, library *config.LibraryState) ([]*gitrepo.ConventionalCommit, error) {
+	var paths []string
+	paths = append(paths, library.SourceRoots...)
+
+	tag := formatTag(library)
+	commits, err := repo.GetCommitsForPathsSinceTag(paths, tag)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get commits for library %s: %w", library.ID, err)
+	}
+	conventionalCommits := []*gitrepo.ConventionalCommit{}
```

Comment:
nit:
```suggestion
	var conventionalCommits []*gitrepo.ConventionalCommit
```

Reply:
Thanks!
TIL: init slice with slice literal creates a non-nil slice of zero length.